docs/docs-beta/docs/dagster-plus/access/rbac/audit-logs.md

```diff
@@ -4,4 +4,51 @@ displayed_sidebar: "dagsterPlus"
 sidebar_position: 4
 ---
 
-# Audit logs
\ No newline at end of file
+# Audit logs
+
+The Dagster Plus audit log enables enterprise organizations to track and attribute changes to their Dagster deployment.
+
+This guide walks through how to access the audit log and details the interactions which are tracked in the audit log.
+
+<details>
+<summary>Prerequisites</summary>
+- A Dagster Plus Pro organization
+- An [Organization Admin](/dagster-plus/access/rbac/user-roles-permissions) role in your Dagster Plus organization
+</details>
+
+## View audit logs
+
+Audit logs are available in the Organization settings pane, which can be accessed by clicking on your user icon on the top right side of the screen and selecting Organization settings in the dropdown menu. Once on the Organization settings pane, select Audit log on the top bar.
+
+Each entry in the audit log indicates when an action was taken, the user who performed the action, the action taken, and the deployment which the action affected. You may click the "Show" button to view additional details for any audit log action.
+
+## Filter the audit log
+
+The "Filter" button on the top left of the Audit log pane can be used to filter the list of logs. You can filter to a combination of user, event type, affected deployment, or time frame.
```

Comment:
```suggestion
The **Filter** button near the top left of the page can be used to filter the list of logs. You can filter to a combination of user, event type, affected deployment, or time frame.
```

- UI components should be bolded
- Suggesting we generally describe where the button is located on the page instead of relation to a (non-labeled) element